services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -391,7 +391,7 @@ public boolean isRunning() {
   }
 
   @Override
-  public synchronized void awaitShutdown() {
+  public void awaitShutdown() {
```

Comment:
OK. Let's revisit the choice between `synchronized` and `AtomicBoolean` later. After reviewing TopicManagementService I have got three other comments. Can you take a look?

- Add `LOG.info(_name + "/TopicManagementService started")` in `TopicManagementService.stop()`. Do the similar thing in `TopicManagementService.start()`
- Set _running to false in `TopicManagementService.stop()`
- Only do `_executor.awaitTermination(...)` in `TopicManagementService.awaitShutdown()`.